components/02-elements/tabs/Tabs.stories.js

```diff
@@ -0,0 +1,32 @@
+import { storiesOf } from '@storybook/vue'
+
+import App from '../../01-globals/app/App.vue'
+import AlpacaTabs from './Tabs.vue'
+
+import tabs from './mocks/tabs.json'
+
+storiesOf('Elements/Tabs', module).add('Default', () => ({
+  components: { App, AlpacaTabs },
+  data: () => ({
+    tabs
+  }),
+  template: `
+    <app>
+      <alpaca-tabs :tabs="tabs.tabs"/>
+    </app>
+  `
+}))
+.add('With icon', () => ({
```

Comment:
I don't see any diffrence between [default](https://alpaca-storybook-priqnl9by.now.sh/?selectedKind=Elements%2FTabs&selectedStory=Default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel) and [with icon](https://alpaca-storybook-priqnl9by.now.sh/?selectedKind=Elements%2FTabs&selectedStory=With%20icon&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel)